dashboard/pages/subscriptions.py

```diff
@@ -1,26 +1,49 @@
 """A dashboard page that lets you sign up for notifications."""
-
+from database import get_all_tags, get_connection
 import streamlit as st
+import boto3
+from os import environ as ENV
+from dotenv import load_dotenv
 
 
-from database import get_all_tags, get_connection
+# Function to create a subscription
+def create_subscription(protocol, endpoint, topic_arn):
+    response = sns_client.subscribe(
+        TopicArn=topic_arn,
+        Protocol=protocol,
+        Endpoint=endpoint
+    )
+    return response
```

Comment:
Could be simplified:

```suggestion
    return sns_client.subscribe(
        TopicArn=topic_arn,
        Protocol=protocol,
        Endpoint=endpoint
    )
```